api/src/controllers/post/administrative-penalty.js

```diff
@@ -198,6 +200,11 @@ exports.createMaster = function (args, res, next, incomingObj, flavourIds) {
  * @returns created lng administrativePenalty record
  */
 exports.createLNG = function (args, res, next, incomingObj) {
+  // Confirm user has correct role to create this type of record.
```

Comment:
This check still has to exist for all `create[flavour]` calls in any Post controllers.  We need to determine that a user has permission to create this type of flavour record. We cannot check the master record as all admins have permission to write on those.